docs/build/html/_sources/methods.rst.txt

```diff
@@ -291,42 +291,56 @@ All other outputs are ``time series x XYZ`` where XYZ is the serialized list of
 with two conductances each: ``NaV`` and ``Kd``, the intrinsic currents would return in the form of a ``time steps x 4`` matrix where the columns would be ordered:
 Kd current from AB, NaV current from AB, Kd current from BC, NaV current from BC.
 
-If ``integrate`` is passed one argument, it interprets this as a matrix of injected currents (in nA). For example ::
+If the ``xolotl`` properties ``I_ext`` and ``V_clamp`` are empty (i.e. ``x.I_ext = []``), the integration proceeds without injected current or clamped voltage.
+Injected current and voltage clamp cannot co-occur.  Setting one of these properties negates the other. If ``I_ext`` is a scalar,
```

Comment:
it's more complicated: if `I_ext` is a Nx1 vector where N is the # of compartments, then a constant current is injected

Reply:
so it's not just about being scalar

Reply:
done